starlette/responses.py

```diff
@@ -243,6 +243,7 @@ def __init__(
         filename: str = None,
         stat_result: os.stat_result = None,
         method: str = None,
+        content_disposition_type: str = "attachment",
```

Comment:
Only "inline" and "attachment" are allowed here, right?

Reply:
Correct